src/client/dfuse/SConscript

```diff
@@ -173,6 +173,22 @@ def configure_fuse(cenv):
     check.Finish()
 
 
+def find_static_fuse(cenv):
+    """Find and copy static fuse library"""
+
+    fuse_prefix = '/usr'
+    if os.path.exists(cenv.subst('$FUSE_PREFIX')):
+        fuse_prefix = cenv.subst('$FUSE_PREFIX')
```

Comment:
FUSE_PREFIX should always be set and exist I'd expect?  If fuse isn't built then it should be set to "/usr" and it would still exist.

Reply:
Actually, it doesn't get defined when USE_INSTALLED is used.  Maybe it should?  But that is a bug for another day